server.js

```diff
@@ -1,30 +1,142 @@
 import express from "express";
 import cors from "cors";
+import booksData from "./data/books.json";
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
+
+// App running on port 8080
 const port = process.env.PORT || 8080;
 const app = express();
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+// Welcome page
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send("Welcome to The Book Server. Enter a path provided in the documentation to get book data");
```

Comment:
When I open "https://project-express-api-on255rqxjq-lz.a.run.app/" in my browser I get a bit stuck on how to continue. It would be nice with some guidance where I don't have to open your repository and read the documentation. Maybe you could include the endpoints here in addition to your documentation, i.e. using express-list-endpoints or manually writing comments?